docker-hub/api/latest.yaml

```diff
@@ -94,6 +94,10 @@ tags:
     x-displayName: Org Settings
     description: |
       The Org Settings API endpoints allow you to manage your organization's settings.
+  - name: repository
```

Comment:
**question:** Shouldn't we call this "Repositories" (plural)? Will make it easier to add onto with more routes.

Reply:
👍 I'll update it